test/specs/lazyload.spec.js

```diff
@@ -111,8 +111,8 @@ describe('LazyLoad', () => {
           </div>
         , div);
 
-        // Differnce between the test above
-        expect(document.querySelector('.test1 .times').textContent).to.equal('3');
+        // Difference between the test above
+        expect(document.querySelector('.test1 .times').textContent).to.equal('2');
```

Comment:
This one should be "2" now because on the second render insertion the `previousVisibility` is the same as `visible`, avoiding the update. Maybe I could add some specs for the new verification as well?